flux-init/init-preprod/kustomization.yaml

```diff
@@ -1,6 +1,39 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- gotk-components-pre.yaml
+- ../base
 - gotk-repo.yaml
 - gotk-sync.yaml
+patches:
```

Comment:
Seems like controller's patch definition is the same as the one in init-prod\gotk-sync.yaml

Reply:
good catch... gonna leave just in this file

Reply:
Have addressed this issue